WordPress/Classes/ViewRelated/Jetpack/Jetpack Restore/Restore Complete/JetpackBackupStatusFailedViewController.swift

```diff
@@ -0,0 +1,40 @@
+import UIKit
+import CocoaLumberjack
+import WordPressShared
+import WordPressUI
+
+class JetpackBackupStatusFailedViewController: BaseRestoreCompleteViewController {
+
+    // MARK: - Initialization
+
+    override init(site: JetpackSiteRef, activity: Activity) {
+        let restoreCompleteConfiguration = JetpackRestoreCompleteConfiguration(
+            title: NSLocalizedString("Status Update Failed", comment: "Title for Jetpack Backup Status Failed screen"),
+            iconImage: .gridicon(.notice),
+            iconImageColor: .error,
+            messageTitle: NSLocalizedString("Hmm, we can't update the status of your backup.", comment: "Title for the Jetpack Backup Status Failed message."),
+            messageDescription: "No need to worry. We'll notify you when your backup is ready.",  // FIXME: Placholder text
```

Comment:
@leandroalonso @emilylaguna (cc: @osullivanchris @zwarm) 
Per our Slack discussion, the message title displays `Hmm, we can't update the status of your backup.`
Trying to come up with a good message description... let me know what you think. Open to suggestions. 👍

Reply:
I think the message is probably ok. But I have some questions about the flows I'll add separately.